client/src/screens/SignUp/SignUp.jsx

```diff
@@ -1,9 +1,62 @@
-import React from 'react'
+import React from 'react';
+import { useState } from 'react';
+import { useNavigate } from "react-router-dom";
+import { Header } from '../../components/Header/Header.jsx'
+import { Form } from '../../components/Form/Form.jsx';
+import { signUp } from "../../services/api/users.js";
 
 export const SignUp = () => {
+  const navigate = useNavigate();
+
+  // define state variables
+  const [formData, setFormData] = useState({
+    email: "",
+    first_name: "",
+    last_name: "",
+    password: "",
+    confirmPassword: "",
+  });
+  const { first_name, last_name, email, password, confirmPassword } = formData;
+
+  // pass header information for Header component
+  const headerTitle = 'Register'
+  const headerText = "In order to find projects and connect with other users, we'll need to create an account."
+  
+ // define functions for screen
+  const handleChange = (event) => {
+    setFormData({
+      ...formData,
+      [event.target.name]: event.target.value,
+    });
+  };
+
+  const handleSubmit = async (event) => {
```

Comment:
since our handleSubmit functions vary depending on what form, we can name them semantically too
for example, here we could call this function `handleSignUp`